### PR TITLE
Summary page - project's current sprint metrics 

### PR DESCRIPTION
**Pull request by vincentcastagna**
- Added new endpoint to retrieve states for a given workItemType, this implied to add a new "HttpBuilder" with good version for this endpoint as available in preview only. 
- Added new class WorkItemStateColor 
- Added metrics properties in ProjectPageViewModel 
- Added tasks to compute WorkItems metrics (summary page metrics) 
- Added loading view for summary view

### CODE REVIEW

File: src/Uno.AzureDevOps/Uno.AzureDevOps.Shared/Client/Data/Team/WorkItemStateColor.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Uno.AzureDevOps.Client
+{
+	public class WorkItemStateColor
```
Comment (andreaSflores18) [inline src/Uno.AzureDevOps/Uno.AzureDevOps.Shared/Client/Data/Team/WorkItemStateColor.cs]: I'd simply name as 'WorkItemState'

Comment (vincentcastagna) [reply src/Uno.AzureDevOps/Uno.AzureDevOps.Shared/Client/Data/Team/WorkItemStateColor.cs]: I asked myself the same but for consistence with API, I named it like it (one day we might need the exact serialized name :) )

File: src/Uno.AzureDevOps/Uno.AzureDevOps.Shared/Presentation/ProjectPageViewModel.cs [hunk 2]
```diff
@@ -173,25 +221,37 @@ public async Task OnNavigatedTo(TeamProjectReference project)
 				_userPreferencesService.SavePreferredProject(project.Id);
 			}
 
-			_project = project;
-			SelectedProjectName = _project.Name;
+			Project = project;
 
+			// SelectedProjectName = _project.Name;
```
Comment (andreaSflores18) [inline src/Uno.AzureDevOps/Uno.AzureDevOps.Shared/Presentation/ProjectPageViewModel.cs]: Remove if not needed

File: src/Uno.AzureDevOps/Uno.AzureDevOps.Shared/Presentation/ProjectPageViewModel.cs [hunk 3]
```diff
@@ -266,12 +328,78 @@ private async Task<List<TeamSettingsIteration>> GetIterations(TeamProjectReferen
...
+
+		private async Task ComputeBugValues()
+		{
+			var bugWorkItems = _workItemsData
+				.Where(w => "Bug".Equals(w.WorkItem.Fields["System.WorkItemType"] as string, StringComparison.OrdinalIgnoreCase))
+				.ToList();
+
+			if (bugWorkItems != null && bugWorkItems.Count > 0)
+			{
+				var bugStates = await _vstsRepository.GetWorkItemTypeStates(Project.Id, bugWorkItems.FirstOrDefault().WorkItemType);
```
Comment (andreaSflores18) [inline src/Uno.AzureDevOps/Uno.AzureDevOps.Shared/Presentation/ProjectPageViewModel.cs]: Could get a `null` after that FirstOrDefault()

File: src/Uno.AzureDevOps/Uno.AzureDevOps.Shared/Presentation/ProjectPageViewModel.cs [hunk 4]
```diff
@@ -266,12 +328,78 @@ private async Task<List<TeamSettingsIteration>> GetIterations(TeamProjectReferen
...
+				.Count();
+			}
+		}
+
+		private async Task ComputePbiValues(List<RichWorkItem> workItems)
+		{
+			var iterationsWorkItems = workItems;
+			if (iterationsWorkItems != null && iterationsWorkItems.Count > 0)
+			{
+				var pbiWorkItemType = iterationsWorkItems.FirstOrDefault().Type.Name;
```
Comment (andreaSflores18) [inline src/Uno.AzureDevOps/Uno.AzureDevOps.Shared/Presentation/ProjectPageViewModel.cs]: Same here

**Pull request merged**